plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/struct/DBStructUtils.java

```diff
@@ -425,6 +425,8 @@ public static String mapTargetDataType(DBSObject objectContainer, DBSTypedObject
             }
             if (dataType != null) {
                 dataKind = dataType.getDataKind();
+                // Datatype caches ignore case, but we probably should use it with the original case
```

Comment:
But usually original <> target, but source. And in this case target type name is used.

Reply:
(About the comment)